tests/schemas/pilot.schema.json

```diff
@@ -262,5 +262,9 @@
     }
   },
   "required": ["name", "standard", "extended", "epic"],
+  "oneOf": [
```

Comment:
@danrs here's where the oneof is - makes logical sense next to the required slots, but double checking there isn't a better way to do it.

Reply:
I think you either do it here or you do it around the fields themselves. This is probably neater as it means less indentation, and if it works I'm fine with it